File: 1118.cpp
#include <iostream>
#include <iomanip>

using namespace std;


int main () {

	double x;
	int a;
	double soma=0,vv=0,media;
	
	while(vv < 2) {
		
		cin >> x;
		
		if ( x < 0 || x >10) {
			cout << "nota invalida" << endl;
		}
		else if ( (x >= 0) || (x <=10)) {
			vv += 1;
			soma += x;
		}
		
	}

	media = (float)(soma/vv);
	
	cout << fixed << setprecision(2) << "media = " << media << endl;
	
	
		
	while (true) {
		
		cout << "novo calculo (1-sim 2-nao)" << endl;
		cin >> a;
		
	if ( a == 2 ) {
		break;		
	}
	else if ( a == 1)
		main();
}
return 0;
}

File: 1118.py
while True:
    total = 0
    lidos = 0

    while lidos < 2:
        valor = float(input())

        if(valor >= 0 and valor <= 10):
            total = total + valor
            lidos += 1
        else:
            print("nota invalida")

    print("media = %.2f" % (total/2))

    o = -1
    while o != 1 and o != 2:
        print("novo calculo (1-sim 2-nao)")
        o = int(input())

    if(o == 2):
        break
